src/elfpy/simulators.py

```diff
@@ -295,6 +298,7 @@ def run_simulation(self, override_dict=None):
                 f"\ninit_time_stretch = {self.market.time_stretch_constant}"
                 "\n-----\n"
             )
+        last_user_action_time = 0
```

Comment:
Can't you pull this variable directly from the analysis dict?

Reply:
ya, but not really needed to store. it's not part of the simulation output, i used it only to do logging when i hadn't seen a trade for a while. it's not used in the final configuration (trades happen often enough). so this should be removed and rolled into proper logging. but it's useful to keep for basic debugging for now.